common/src/main/java/com/wynntils/features/inventory/InventoryEmeraldCountFeature.java

```diff
@@ -39,6 +39,9 @@ public class InventoryEmeraldCountFeature extends Feature {
     @RegisterConfig
     public final Config<EmeraldCountType> emeraldCountType = new Config<>(EmeraldCountType.TEXTURE);
 
+    @RegisterConfig
+    public final Config<TextDisplaySide> textDisplaySide = new Config<>(TextDisplaySide.LEFT);
```

Comment:
Why does this whole change only work with text? I don't see any reason not to move the textured one as well.

Reply:
The texture one doesn't work be cause the "effect" things is above it, so it cuts out